docs/guide/custom-ui.md

```diff
@@ -398,15 +398,25 @@ Before you override an editor modal, consider [adding a custom schema field type
 
 ## Adding custom context menu items
 
-Apostrophe offers a context menu that can be used to carry out certain operations on a document, such as 'preview', 'duplicate', and so on. We can add custom context menu items from within any module, targeting any Vue component that implements `AposModal`. For an example of this, see the [code for the draft sharing modal](https://github.com/apostrophecms/apostrophe/blob/main/modules/%40apostrophecms/modal/ui/apos/components/AposModalShareDraft.vue). The menu registration should happen in the initialization phase. It is important to note that context menu operations will appear for all documents, even if added by a module associated with a specific type of document. However, take note of the various options below to limit when they appear.
+Apostrophe offers a context menu that can be used to carry out certain operations on a document, such as 'preview', 'duplicate', and so on. You can add custom context menu items from within any module.
```

Comment:
Not new, but shouldn't these be double quotes?

Reply:
A screenshot is in order here for... context.

Reply:
Yes to the double quotes from looking at the rest of the document. Added.
I moved the screenshot up tp just below this paragraph.